modules/hcp-dr-oadp-restore-new-mgmt.adoc

```diff
@@ -0,0 +1,302 @@
+----
+
+... Start the reconciliation of the `NodePool` resource by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <restore_management_kubeconfig> \
+  patch nodepool -n <hosted_cluster_namespace> <node_pool_name> \
+  --type json \
+  -p '[{"op": "replace", "path": "/spec/pausedUntil", "value": "false"}]'
+----
+
+... Verify that the hosted cluster is reporting that the hosted control plane is available by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <restore_management_kubeconfig> get hostedcluster
+----
+
+... Verify that the hosted cluster is reporting that the cluster operators are available by running the following command:
++
+[source,terminal]
+----
+$ oc get co --kubeconfig <hosted_cluster_kubeconfig>
+----
+
+.. Start the reconciliation of the Agent provider resources that you paused during backing up of the control plane workload:
+
+... Start the reconciliation of the `AgentCluster` resource by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <restore_management_kubeconfig> \
+  annotate agentcluster -n <hosted_control_plane_namespace>  \
+  cluster.x-k8s.io/paused- --overwrite=true --all
+----
+
+... Start the reconciliation of the `AgentMachine` resource by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <restore_management_kubeconfig> \
+  annotate agentmachine -n <hosted_control_plane_namespace>  \
+  cluster.x-k8s.io/paused- --overwrite=true --all
+----
+
+... Start the reconciliation of the `Cluster` resource by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <restore_management_kubeconfig> \
+  annotate cluster -n <hosted_control_plane_namespace> \
+  cluster.x-k8s.io/paused- --overwrite=true --all
+----
+
+.. Verify that the node pool is working as expected by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <restore_management_kubeconfig> \
+  get nodepool -n <hosted_cluster_namespace> 
+----
++
+.Example output
+[source,terminal]
+----
+NAME       CLUSTER    DESIRED NODES   CURRENT NODES   AUTOSCALING   AUTOREPAIR   VERSION   UPDATINGVERSION   UPDATINGCONFIG   MESSAGE
+hosted-0   hosted-0   3               3               False         False        4.17.11   False             False
+----
+
+.. Optional: To ensure that no conflicts exist and that the new management cluster has continued functionality, remove the `HostedCluster` resources from the backup management cluster by completing the following steps:
+
+... In the management cluster that you backed up from, in the `ClusterDeployment` settings, set the `spec.preserveOnDelete` parameter to `true` by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <backup_management_kubeconfig> patch \
+  -n <hosted_control_plane_namespace> \
+  ClusterDeployment/<hosted_cluster_name> -p \
+  '{"spec":{"preserveOnDelete":'true'}}' \
```

Comment:
Is this syntax correct? I ask because I see what I would've expected on line 267. `--type=json`

Reply:
@CrystalChun Can you answer Joe's question here?

Reply:
Merge can be used for patching as well. I just ran this on my cluster

```sh
$ oc patch -n spoke-assisted-test-cluster clusterdeployment/assisted-test-cluster -p '{"spec":{"preserveOnDelete": 'true'}}' --type=merge
clusterdeployment.hive.openshift.io/assisted-test-cluster patched
$ oc get -n spoke-assisted-test-cluster clusterdeployment assisted-test-cluster -ojsonpath='{.spec.preserveOnDelete}'
true
```

Reply:
Thanks for confirming that `type=merge` works!